addons/repair/views/repair_views.xml

```diff
@@ -193,8 +200,10 @@
                             </group>
                         </group>
                     </page>
-                    <page string="Notes" name="notes">
+                    <page string="Repair Notes">
```

Comment:
Removing the name attribute makes it difficult to develop extensions for the module. Any specific reasons for this removal?

Reply:
Hi @marcelsavegnago, thanks for the notification! It's a mistake that will be corrected in the next update of the repair module.